README.md

```diff
@@ -1,12 +1,9 @@
-![icon](https://i.imgur.com/3rrt5b9.png)
+![icon](https://i.imgur.com/HHa3Tfs.png)
 
 
 ## Resizable
-![resizable](https://i.imgur.com/FQZHqJm.png)
+![resizable](https://i.imgur.com/gKYy5UE.png)
+![resizable](https://i.imgur.com/N33jsML.png)
 
 ## Fixed
-![resizable](https://i.imgur.com/5rb6QUN.png)
-
-Big thanks to Melky for having the patience to deal with my garbage Git skills.
-
-Thank you to sgfost and EXpoZuR for creating the such great skins, having such great sprites, and giving me the inspiration to get this done.
```

Comment:
Why did you get rid of credit, I see you are still using icons from reaper-chroma?

Reply:
Also there is a .psd (specifically equipment_guide_prices.psd) file included which should not be there